src/main/java/frc/robot/commands/DriveWithGamepadCommand.java

```diff
@@ -27,7 +27,7 @@ public void initialize() {}
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-    m_drive.arcadeDrive(-m_controller.getLeftY(),m_controller.getRightX());
+    m_drive.arcadeDrive(-m_controller.getLeftY(),-m_controller.getRightX());
```

Comment:
We should look into what is off for a few minutes on thursday. I wonder if the we need to invert the left side instead of the right side. IIRC The gearbox on the 2020 robot has 3 stages, and this years robot has 2 stages. So the 1 less stage would cause the side we invert to be different.t

We need to make sure that the positive forward values make the robot turn forward, and positive rotation values should rotate the robot clockwise per WPIDocs for the DifferentialDrive class.  
```
zRotation - The robot's rotation rate around the Z axis [-1.0..1.0]. Clockwise is positive.
```

When we move the joystick to the right (+X axis) we want to have the robot turn clockwise, but we are negating it for it to turn clockwise.  

If we don't get things just right, this will cause problems for the Trajectory stuff that we are also working on, as we can't trust that the robot is configured right.